packages/documentation/src/stories/components/select/select.stories.ts

```diff
@@ -104,7 +104,7 @@ const meta: MetaComponent = {
     multipleSize: {
       name: 'Multiple Size',
       description:
-        'When set to a number larger than 0, will set the number of display option rows.<div className="text-danger">Note: not all browser will respect this setting.</div>',
+        'When set to a number larger than 0, will set the number of display option rows.<div class="colored-text-error">Note: not all browser will respect this setting.</div>',
```

Comment:
```suggestion
        'When set to a number larger than 0, will set the number of display option rows.<div class="banner banner-sm banner-danger">Note: not all browser will respect this setting.</div>',
```
This would fit better with all the other messages we add here and there.
And we would not need to implement a custom variable for that, nor being dependant on the `post.$error` variable.

![image](https://github.com/user-attachments/assets/4f0f488e-2d94-4316-a037-4313f05092ec)